catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+  links:
+    - url: "https://github.com/ichintanjoshi/frontend-app-shell"
+      title: "frontend-app-shell"
+      icon: "GitHub"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+    openedx.org/release: "main"
```

Comment:
You should not put in this value by default, it should only go in if the repo is going to be tagged for releases,  it should match the value in the `openedx.yaml` file in the repo if it exists. If it doesn't exist, then this annotation should not be added.

Reply:
@feanil  have removed